Output_Formatter/NMEA_format.cpp

```diff
@@ -561,9 +561,14 @@ void format_NMEA_string( const output_data_t &output_data, string_buffer_t &NMEA
 
 #if USE_LARUS_NMEA_EXTENSIONS
 
+#if  HORIZON_DATA_SECRET == 1
   // aircraft attitude
+  format_PLARA( ZERO, ZERO, output_data.euler.y, next);
```

Comment:
Why not just skip sending this sentence in case horizon output is disabled?

Reply:
I think disabling shall be controlled by a function parameter, so that the software using sw_sensor_algorithms can disable and activate the horizon feature in a more dynamic way.  

With this solution we would need to release always two sensor firmware versions. One activated (for competitions) and one deactivated.

Reply:
Eventually people still want/need the true heading,
even without the horizon.
Another option would be to use a configuration parameter
to switch the horizon on/off.

Reply:
Do we want to provide a configuration option e.g. via a sd-card parameter which disables the attitude NMEA/CAN output in the sensor firmware. I would prefer to have a function parameter so that we can potentially have the option to implement disabling the attitude output during runtime. 

I think we should also cleanup the following to improve readability:
Can we remove all non Larus Sentences and stick only to the NMEA protocol definition from here: https://github.com/larus-breeze/doc_larus/blob/master/documentation/Larus_NMEA_Protocol.md ?   

This would mean to remove and cleanup the following defines:
#if USE_POV // the OpenVario way to do it ...     
#if USE_PTAS1  
#if USE_MWV
#if WITH_DENSITY_DATA

#if USE_LARUS_NMEA_EXTENSIONS  Remove the define or is there really still a use case to disable LARUS_NMEA extensions?

Reply:
Sure, we shall clean up all that stuff.